### DISCUSSION TITLE
Authentication with google provider does not give option to sign in with another account

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#448] [General]
I implemented login with the google provider, I did the deploy in the replit, when I access the smart phone, when opening a tab to choose the google account, it is closed quickly, and the account that is currently logged in the browser is used, no giving you the option to sign in with another account.

P.s. on desktop it works without problem.

I don't know if in the example with the github provider the same happens.

**Comment by FeodorFitsner** [comment]
I'm not sure if there is a parameter in authorization flow that allows to choose an account.
Try cleaning cookies or use private browsing to see how it works.

**Reply by nascin** [reply]
with private browsing it worked fine, but for my use case, I need the user to choose an account, it won't necessarily be the same one he's logged into the browser.

**Reply by FeodorFitsner** [reply]
OK, I found this: https://stackoverflow.com/a/39859935/1435891

It's impossible to specify this parameter for built-in GoogleOAuthProvider, but you can configure a custom provider: https://flet.dev/docs/guides/python/authentication#configuring-a-custom-oauth-provider by copying most of the settings from https://github.com/flet-dev/flet/blob/main/sdk/python/flet/auth/providers/google_oauth_provider.py

Let me know if that `prompt` parameter worked and I'll add it to the core SDK.
